.github/workflows/test.yml

```diff
@@ -35,25 +38,14 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-mix-${{ env.cache-name }}-
 
-      - name: Cache compiled build
```

Comment:
This seemed to be causing a BEAM version mismatch, I suppose because of the two versions (1.12/OTP24 and 1.16/OTP26) being tested. It didn't seem to appreciably increase CI run time (it's still <50s)